lib/MySQL_Monitor.cpp

```diff
@@ -5816,5 +5891,1340 @@ bool DNS_Cache::empty() const {
 	return result;
 }
 
+#define NEXT_IMMEDIATE(new_st) do { async_state_machine_=new_st; goto __again; } while (0)
+
+short MySQL_Monitor_State_Data::next_event(MDB_ASYNC_ST new_st, int status) {
+
```

Comment:
Same kind of empty line as mentioned in other comments.

Reply:
It's done intentionally to avoid code clutter.